.github/workflows/upload-release-asset.yml

```diff
@@ -0,0 +1,37 @@
+on:
+  release:
+    types: [created, edited]
```

Comment:
`edited` might be tricky. If `edited` is triggered this workflow again, any text or description change might result in a new file. 

Do we really need to trigger it when a release is updated? Generally, we don't update tag version. That means files will not change in any case on a release.

Reply:
Oh, if that is the case, you are right we don't need it on 'edited'. I assumed that we might have scenarios where release might have file changes.